core/src/main/scala/com/microsoft/azure/synapse/ml/param/DataFrameParam.scala

```diff
@@ -116,8 +115,11 @@ trait DataFrameEquality extends Serializable {
   */
 class DataFrameParam(parent: Params, name: String, doc: String, isValid: DataFrame => Boolean)
   extends ComplexParam[DataFrame](parent, name, doc, isValid)
-    with ExternalPythonWrappableParam[DataFrame] with ParamEquality[DataFrame] with DataFrameEquality
-    with ExternalDotnetWrappableParam[DataFrame] {
+    with ParamEquality[DataFrame]
+    with DataFrameEquality
+    with ExternalPythonWrappableParam[DataFrame]
```

Comment:
Might want to make a simple mixer class like ExternalWrappableParam which just mixes in the other 3 traits for simple interfaces